CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Fix legacy colour on input component ([PR #1487](https://github.com/alphagov/govuk_publishing_components/pull/1487))
```

Comment:
Do we need this in the changelog?

Reply:
I think it's valuable for services still having one of the compatibility flags on